Ladybird/WebWorker/CMakeLists.txt

```diff
@@ -2,9 +2,7 @@ include(fontconfig)
 
 set(WEBWORKER_SOURCE_DIR ${LADYBIRD_SOURCE_DIR}/Userland/Services/WebWorker)
 
-set(CMAKE_AUTOMOC OFF)
-set(CMAKE_AUTORCC OFF)
-set(CMAKE_AUTOUIC OFF)
+set_target_properties(webcontent PROPERTIES AUTOMOC OFF AUTORCC OFF AUTOUIC OFF)
```

Comment:
you'll need this on the webworker library, due to macOS using a case-insenstive filesystem 🙃 . copy pasta from WebContent?

Reply:
Yeah, this was totally a copy paste mistake :sweat_smile:. MacOS CI seems happy now.